layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -235,6 +235,7 @@
   "cd" 'spacemacs/close-compilation-window)
 (with-eval-after-load 'compile
   (define-key compilation-mode-map "r" 'recompile)
+  (define-key evil-motion-state-map "gf" 'find-file-at-point)
```

Comment:
Updated https://github.com/syl20bnr/spacemacs/pull/7712/commits/12517cf695043b49ad7cc3c489af37cb0c270ab6
Is this one fine?